src/app/components/SearchForm/SearchForm.jsx

```diff
@@ -61,15 +62,13 @@ class SearchForm extends React.Component {
                       onChange={this.onFieldChange}
                       value={this.state.searchField}
                     >
-                      <option value={this.props.allowedFields.kw} defaultValue>
-                        Keyword
-                      </option>
-                      <option value={this.props.allowedFields.ti}>
-                        Title
-                      </option>
-                      <option value={this.props.allowedFields.au}>
-                        Author
-                      </option>
+                      {this.props.allowedFields.map((element, key) => {
```

Comment:
I might change the `element` variable name to something different since you can pass HTML elements around in jsx and that could be confusing.

Reply:
Fixing to use `field` instead of `element`